docs/services-guide.md

```diff
@@ -4,772 +4,133 @@ toc: true
-use with a Wordpress Deployment. However, this involved configuring multiple
-Kubernetes objects to get a fully functioning Wordpress instance securely
-deployed.
-
-Stacks can also be used to simplify app management and automate operations. Our
-next guide shows how an App Stack can automate most of the steps covered in this
-guide and be run from a dedicated control plane that: (a) dynamically provisions
-the target cluster, (b) provisions the managed services, and (c) deploys the app
-itself with secure connectivity.
-
-App Stacks simplify operations for an app by moving the steps covered in this guide into a Kubernetes controller that owns an app CRD (custom resource definition) with a handful of settings required to deploy a new app instance, complete with the managed services it depends on.
-
-## Next Steps
-* [Crossplane Stacks Guide][stack-user-guide] to deploy the same Wordpress instance with a
-  single yaml file, using the [portable Wordpress App Stack][stack-wordpress].
-* [Extend a Stack][stack-developer-guide] to add more cloud services to:
-  [stack-gcp][stack-gcp], [stack-aws][stack-aws], or [stack-azure][stack-azure].
-* [Build a new Stack][stack-developer-guide] to add more cloud providers or
-  independent cloud services.
-
-If you have any questions, please drop us a note on [Crossplane Slack][join-crossplane-slack] or [contact us][contact-us]!
-
-## References
-### Concepts
-* [Crossplane Concepts][crossplane-concepts]
-* [Claims][concept-claim]
-* [Classes][concept-class]
-* [Portable Classes][concept-portable-class]
-* [Default Classes][concept-default-class]
-* [Workloads][concept-workload]
-* [Stacks][concept-stack]
-* [Stacks Design][stack-design]
-* [Stacks Manager][stack-manager]
-* [Stacks Registry][stack-registry]
-* [Stack Install Flow][stack-install-docs]
-* [Stack Package Format][stack-format-docs]
-
-### Getting Started
-* [Install Crossplane][install-crossplane]
-* [Install Provider Stacks][install-provider-stacks]
-* [Cloud Provider Credentials][cloud-provider-creds]
-* [Crossplane CLI][crossplane-cli]
-* [Crossplane CLI Docs][crossplane-cli-docs]
-
-**GCP**
-* [GCP Services Guide][gcp-services-guide]
-* [GCP Stack][stack-gcp]
-* [GCP Docs][gcp-docs]
-
-**AWS**
-* [AWS Services Guide][aws-services-guide]
-* [AWS Stack][stack-aws]
-* [AWS Docs][aws-docs]
-
-**AWS**
-* [Azure Services Guide][azure-services-guide]
-* [Azure Stack][stack-azure]
-* [Azure Docs][azure-docs]
-
-### Using and Building Stacks
-* [Stacks Guide][stack-user-guide]
-* [Stacks Developer Quick Start][stack-quick-start]
-* [Stacks Developer Guide][stack-developer-guide]
-
-### Kubernetes
-* [Kubernetes Concepts][kubernetes-concepts]
-* [Kubernetes Docs][kubernetes-docs]
-* [kubectl docs][kubectl-docs]
+This guide is an overview of enabling cloud service 
+provisioning on an existing Kubernetes target cluster, 
+including how to integrate Crossplane with existing cloud networking configurations 
```

Comment:
nitpick: word-wrap looks like it might be acting strangely on these lines here. I could be wrong though